css/properties/outline.json

```
@@ -64,6 +64,57 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "invert": {
+          "__compat": {
+            "description": "<code>invert</code>",
```

Two things here:

1. No description is needed, since it's just the name of the feature
2. It's good to have a spec_url here

```suggestion
            "spec_url": "https://drafts.csswg.org/css-ui/#valdef-outline-color-invert",
```